src/main/java/org/jitsi/utils/logging2/Logger.java

```diff
@@ -0,0 +1,182 @@
+    /**
+     * Check if a message with a TRACE level would actually be logged by this
+     * logger.
+     * <p>
+     * @return true if the TRACE level is currently being logged
+     */
+    boolean isTraceEnabled();
+
+    /**
+     * Log a TRACE message.
+     * <p>
+     * If the logger is currently enabled for the TRACE message
+     * level then the given message is forwarded to all the
+     * registered output Handler objects.
+     * <p>
+     * @param msg The message to log
+     */
+    void trace(Object msg);
+
+    /**
+     * Check if a message with a DEBUG level would actually be logged by this
+     * logger.
+     * <p>
+     * @return true if the DEBUG level is currently being logged
+     */
+    boolean isDebugEnabled();
+
+    /**
+     * Log a DEBUG message.
+     * <p>
+     * If the logger is currently enabled for the DEBUG message
+     * level then the given message is forwarded to all the
+     * registered output Handler objects.
+     * <p>
+     * @param msg The message to log
+     */
+    void debug(Object msg);
+
+    /**
+     * Check if a message with an INFO level would actually be logged by this
+     * logger.
+     *
+     * @return true if the INFO level is currently being logged
+     */
+    boolean isInfoEnabled();
+
+    /**
+     * Log a INFO message.
+     * <p>
+     * If the logger is currently enabled for the INFO message
+     * level then the given message is forwarded to all the
+     * registered output Handler objects.
+     * <p>
+     * @param msg The message to log
+     */
+    void info(Object msg);
+
+    /**
+     * Check if a message with a WARN level would actually be logged by this
+     * logger.
+     * <p>
+     * @return true if the WARN level is currently being logged
+     */
+    boolean isWarnEnabled();
+
+    /**
+     * Log a WARN message.
+     * <p>
+     * If the logger is currently enabled for the WARN message
+     * level then the given message is forwarded to all the
+     * registered output Handler objects.
+     * <p>
+     * @param msg The message to log
+     */
+    void warn(Object msg);
+
+    /**
+     * Log a message, with associated Throwable information.
+     * <p>
+     * @param msg   The message to log
+     * @param t  Throwable associated with log message.
+     */
+    void warn(Object msg, Throwable t);
+
+    /**
+     * Log a ERROR message.
+     * <p>
+     * If the logger is currently enabled for the ERROR message
+     * level then the given message is forwarded to all the
+     * registered output Handler objects.
+     * <p>
+     * @param msg The message to log
+     */
+    void error(Object msg);
+
+    /**
+     * Log a message, with associated Throwable information.
+     * <p>
+     * @param msg   The message to log
+     * @param t  Throwable associated with log message.
+     */
+    void error(Object msg, Throwable t);
+
+    /**
+     * Set logging level for all handlers to ERROR
+     */
+    void setLevelError();
```

Comment:
Do you think we need all these, or were they just present in the original? I'd remove them and use `setLevel(Level)`

Reply:
Up to you